cmp/store/StoreFilterField.js

```diff
@@ -89,9 +92,6 @@ StoreFilterField.propTypes = {
      */
     includeFields: PT.arrayOf(PT.string),
 
-    /** Optional model for value binding - see comments on the `bind` prop for details. */
```

Comment:
FWIW I still prefer having these alpha-sorted.  Otherwise we rarely have a consistent ordering, and it makes it hard to scan for and confirm the existence of a prop when looking @ the source during app development.